src/test/suite/completion.test.ts

```diff
@@ -84,47 +95,107 @@ describe("Should do completion", () => {
     );
   });
   it("should prefer the popup when only popup is visible and there is no inline suggestion", async () => {
-    (await openADocWith("cons")).makeAChange("o");
+    await openADocWith("cons");
+
+    await makeAChange("o");
 
     await triggerPopupSuggestion();
 
     await triggerInline();
 
-    await triggerSelectionAppetence();
+    await emulationUserInteraction();
+
+    await triggerSelectionAcceptance();
 
     assertTextIsCommitted("console");
   });
   it("should prefer an inline when both popup and inline are visible", async () => {
-    (await openADocWith("cons")).makeAChange("o");
+    await openADocWith("cons");
+
+    await makeAChange("o");
 
     await triggerPopupSuggestion();
 
     mockInlineResponse();
 
     await triggerInline();
 
-    await triggerSelectionAppetence();
+    await triggerSelectionAcceptance();
 
     assertTextIsCommitted("console.log");
 
     await makeAndAssertFollowingChange();
   });
   it("should skip completion request on midline invalid position", async () => {
-    await openADocWith(" s");
+    await openADocWith("console s");
 
-    await moveToStartOfLinePosition();
+    await moveLeftBy(2);
```

Comment:
this test is not clear to me, because I don't know where is the cursor to begin with - so I don't know what's the meaning of moving it by 2